### PR TITLE
[4.6] chore(a11y): generate wai aria articles with the latest a11y specs

### PR DESCRIPTION
**Pull request by xristianstefanov**

### CODE REVIEW

File: components/pdfviewer/accessibility/wai-aria-support.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+slug: pdfviewer-wai-aria-support 
+position: 50 
+---
+
+# Blazor PDFViewer Accessibility
+
+@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
+
+
+
```
Comment (dimodi) [inline components/pdfviewer/accessibility/wai-aria-support.md:15]:
```suggestion
```

Comment (dimodi) [inline components/pdfviewer/accessibility/wai-aria-support.md:15]:
```suggestion
```

File: components/pdfviewer/accessibility/wai-aria-support.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+Out of the box, the Telerik UI for Blazor PDFViewer provides extensive accessibility support and enables users with disabilities to acquire complete control over its features.
+
+
+The PDFViewer is compliant with the [Web Content Accessibility Guidelines (WCAG) 2.1  AA](https://www.w3.org/TR/WCAG21/) standards](https://www.w3.org/TR/WCAG21/) and [Section 508](http://www.section508.gov/) requirements, follows the [Web Accessibility Initiative - Accessible Rich Internet Applications (WAI-ARIA)](https://www.w3.org/WAI/ARIA/apg/) best practices for implementing the [keyboard navigation](#keyboard-navigation) for its `component` role, provides options for managing its focus and is tested against the most popular screen readers.
```
Comment (dimodi) [inline components/pdfviewer/accessibility/wai-aria-support.md:19]: Split to a list for better readability.

File: components/pdfviewer/accessibility/wai-aria-support.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+## WAI-ARIA
+
+
```
Comment (dimodi) [inline components/pdfviewer/accessibility/wai-aria-support.md:23]:
```suggestion
```

File: components/pdfviewer/accessibility/wai-aria-support.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+This section lists the selectors, attributes, and behavior patterns supported by the component and its composite elements, if any.
+
+
```
Comment (dimodi) [inline components/pdfviewer/accessibility/wai-aria-support.md:26]:
```suggestion
```

File: components/pdfviewer/accessibility/wai-aria-support.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+| `.k-pdfviewer .k-canvas` | `tabindex=0` | Defines the focusable page container element. |
+|  | `aria-label` | Describes the purpose of the focusable container. Translatable message. |
+|  | `role=document` | Defines that content should be evaluated in reader mode by assistive technologies. |
+| `.k-pdfviewer .k-toolbar .k-button:has(.k-svg-i-search, .k-i-search)` | `aria-haspopup=dialog` | Describes that the Search tool button opens a dialog element. |
+| `.k-pdfviewer .k-canvas .k-search-panel` | `role=dialog` | Describes the role of the Search panel. |
+|  | `aria-label` | Translatable message, same label as the one, used to describe the Toolbar Search tool. |
+
+## Section 508
+
+
```
Comment (dimodi) [inline components/pdfviewer/accessibility/wai-aria-support.md:42]:
```suggestion
```

File: components/pdfviewer/accessibility/wai-aria-support.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+The PDFViewer is fully compliant with the [Section 508 requirements](http://www.section508.gov/).
+
+## Testing
+
+
```
Comment (dimodi) [inline components/pdfviewer/accessibility/wai-aria-support.md:47]:
```suggestion
```

File: components/pdfviewer/accessibility/wai-aria-support.md [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+The PDFViewer has been extensively tested automatically with [axe-core](https://github.com/dequelabs/axe-core) and manually with the most popular screen readers.
+
+> To report any accessibility issues, contact the team through the [Telerik Support System](https://www.telerik.com/account/support-center).
+
+### Screen Readers
+
+
```
Comment (dimodi) [inline components/pdfviewer/accessibility/wai-aria-support.md:54]:
```suggestion
```

File: components/pdfviewer/accessibility/wai-aria-support.md [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+The PDFViewer has been tested with the following screen readers and browsers combinations:
```
Comment (dimodi) [inline components/pdfviewer/accessibility/wai-aria-support.md:55]: When a noun is used as an adjective, it must be in singular.

```suggestion
The PDFViewer has been tested with the following screen reader and browser combinations:
```

File: components/pdfviewer/accessibility/wai-aria-support.md [hunk 9]
```diff
@@ -0,0 +1,69 @@
...
+
+| Environment | Tool |
```
Comment (dimodi) [inline components/pdfviewer/accessibility/wai-aria-support.md:57]:
```suggestion
| Browser | Screen Reader |
```

File: components/pdfviewer/accessibility/wai-aria-support.md [hunk 10]
```diff
@@ -0,0 +1,69 @@
...
+| ----------- | ---- |
+| Firefox | NVDA |
+| Chrome | JAWS |
+| Microsoft Edge | JAWS |
+
+
+
```
Comment (dimodi) [inline components/pdfviewer/accessibility/wai-aria-support.md:64]:
```suggestion
```

**Review by dimodi — APPROVED** [review]
Use a list after "The PDFViewer is compliant with:"

**Comment by xristianstefanov** [comment]
> Use a list after "The PDFViewer is compliant with:"

Thank you for your suggestions. We will certainly take them into account for future consideration, particularly after merging this PR as these articles are automatically generated for all technologies.

File: components/pdfviewer/accessibility/wai-aria-support.md [hunk 11]
```diff
@@ -0,0 +1,69 @@
...
+The PDFViewer is compliant with the [Web Content Accessibility Guidelines (WCAG) 2.1  AA](https://www.w3.org/TR/WCAG21/) standards](https://www.w3.org/TR/WCAG21/) and [Section 508](http://www.section508.gov/) requirements, follows the [Web Accessibility Initiative - Accessible Rich Internet Applications (WAI-ARIA)](https://www.w3.org/WAI/ARIA/apg/) best practices for implementing the [keyboard navigation](#keyboard-navigation) for its `component` role, provides options for managing its focus and is tested against the most popular screen readers.
+
+## WAI-ARIA
+
+
+This section lists the selectors, attributes, and behavior patterns supported by the component and its composite elements, if any.
+
+
+The PDFViewer component contains two inner elements - a toolbar and a page container.
```
Comment (yordan-mitev) [inline components/pdfviewer/accessibility/wai-aria-support.md:27]:
```suggestion
The PDFViewer component contains two inner elements—a toolbar and a page container.
```

File: components/pdfviewer/accessibility/wai-aria-support.md [hunk 12]
```diff
@@ -0,0 +1,69 @@
...
+
+[ToolBar accessibility specification]({{toolbar_a11y_link}})
```
Comment (yordan-mitev) [inline components/pdfviewer/accessibility/wai-aria-support.md:29]: Is this link related to the table below? I'm struggling to understand why it's placed like this between the previous sentence and the table below. Consider using this anchor as a part of a complete sentence.

File: components/pdfviewer/accessibility/wai-aria-support.md (hunk 11 above)
Comment (xristianstefanov) [reply components/pdfviewer/accessibility/wai-aria-support.md:27]: This PR only triggers the autogeneration of this article. Your suggestion will be taken into account after we merge this.

File: components/pdfviewer/accessibility/wai-aria-support.md (hunk 12 above)
Comment (xristianstefanov) [reply components/pdfviewer/accessibility/wai-aria-support.md:29]: This PR only triggers the autogeneration of this article and causes placeholders in some places. It will be fixed.
